packages/datetime/src/components/date-picker3/datePickerUtils.ts

```diff
@@ -14,10 +14,22 @@
  * limitations under the License.
  */
 
+import { Months } from "../../common";
 import { getFormattedDateString } from "../../common/dateFormatProps";
 import { measureTextWidth } from "../../common/utils";
 
-import { getDefaultMaxDate, getDefaultMinDate } from "./datePickerCore";
```

Comment:
These were inlined since the other utilities from `datePickerCore.tsx` are no longer needed.